web/pgadmin/static/js/components/FormComponents.jsx

```diff
@@ -1132,76 +1129,79 @@ FormNote.propTypes = {
+  {
+    theme
+  }
+) => ({
+  '&.FormFooter-root': {
     padding: theme.spacing(0.5),
     position: 'absolute',
     bottom: 0,
     left: 0,
     right: 0,
     zIndex: 10,
   },
-  container: {
+  '&.FormFooter-container': {
     borderWidth: '1px',
     borderStyle: 'solid',
     borderRadius: theme.shape.borderRadius,
     padding: theme.spacing(0.5),
     display: 'flex',
     alignItems: 'center',
     minHeight: '36px',
-  },
-  containerSuccess: {
-    borderColor: theme.palette.success.main,
-    backgroundColor: theme.palette.success.light,
-  },
-  iconSuccess: {
-    color: theme.palette.success.main,
-  },
-  containerError: {
-    borderColor: theme.palette.error.main,
-    backgroundColor: theme.palette.error.light,
-  },
-  iconError: {
-    color: theme.palette.error.main,
-  },
-  containerInfo: {
-    borderColor: theme.palette.primary.main,
-    backgroundColor: theme.palette.primary.light,
-  },
-  iconInfo: {
-    color: theme.palette.primary.main,
-  },
-  containerWarning: {
-    borderColor: theme.palette.warning.main,
-    backgroundColor: theme.palette.warning.light,
-  },
-  iconWarning: {
-    color: theme.palette.warning.main,
-  },
-  message: {
-    color: theme.palette.text.primary,
-    marginLeft: theme.spacing(0.5),
-  },
-  messageCenter: {
-    color: theme.palette.text.primary,
-    margin: 'auto',
-  },
-  closeButton: {
-    marginLeft: 'auto',
+    '&.FormFooter-containerError': {
+      borderColor: theme.palette.error.main,
+      backgroundColor: theme.palette.error.light,
+      '& .FormFooter-iconError': {
+        color: theme.palette.error.main,
+      },
+    },
+    '&.FormFooter-containerSuccess': {
+      borderColor: theme.palette.success.main,
+      backgroundColor: theme.palette.success.light,
+      '& .FormFooter-iconSuccess': {
+        color: theme.palette.success.main,
+      },
+    },
+    '&.FormFooter-containerInfo': {
+      borderColor: theme.palette.primary.main,
+      backgroundColor: theme.palette.primary.light,
+      '& .FormFooter-iconInfo': {
+        color: theme.palette.primary.main,
+      },
+    },
+    '&.FormFooter-containerWarning': {
+      borderColor: theme.palette.warning.main,
+      backgroundColor: theme.palette.warning.light,
+      '& .FormFooter-iconWarning': {
+        color: theme.palette.warning.main,
+      },
+    },
+    '& .FormFooter-message': {
+      color: theme.palette.text.primary,
+      marginLeft: theme.spacing(0.5),
+    },
+    '& .FormFooter-messageCenter': {
+      color: theme.palette.text.primary,
+      margin: 'auto',
+    },
+    '& .FormFooter-closeButton': {
+      marginLeft: 'auto',
+    },
   },
 }));
 
 /* The form footer used mostly for showing error */
 export function FormFooterMessage({style, ...props}) {
-  const classes = useStylesFormFooter();
-
   if (!props.message) {
     return <></>;
   }
   return (
-    <Box className={classes.root} style={style}>
+    <StyledBox className='FormFooter-root' style={style}>
```

Comment:
no need to add class to root if it is not conditional

Reply:
Done.